src/misc/setup.cpp

```diff
@@ -867,6 +867,16 @@ Property* Section_prop::Get_prop(int index)
 	return nullptr;
 }
 
+Property* Section_prop::Get_prop(const std::string& propname)
+{
+	for (Property* property : properties) {
+		if (property->propname == propname) {
+			return property;
+		}
+	}
+	return nullptr;
```

Comment:
```suggestion
    const auto it = std::ranges::find_if(properties, [propname](const auto* prop) { return prop->propname == propname); };
    return it != properties.end() ? *it : nullptr;
```

Reply:
I think the loop is much easier to read.

Reply:
I tend to agree; we don't get brownie points for using STL to the max. Let's optimise for readability.